p4-16/psa/PSA.mdk

```diff
@@ -912,10 +890,6 @@ which port to send the cloned packet to after egress processing. The
 instance, it can be a port to control CPU or a port to the next hop
 router.
 
-~ TBD
-Decide how to add other metadata to the cloned packet (see [#sec-open-metadata]).
-~
-
```

Comment:
It seems like it should be covered by whatever we come up with for passing metadata with packets, which is a prominent issue in the open issues section.